src/client/menu/menu.c

```diff
@@ -2915,9 +2915,37 @@ Mods_MenuInit(void)
     int currentmod;
     int x = 0;
     int y = 0;
+	char modname[MAX_QPATH]; //TG626
 
     Mods_NamesInit();
-
+	
+	// create array of bracketed display names from folder names - TG626
+	char **displaynames=malloc(sizeof(*displaynames) * nummods);
+	
+	for (int i=0; i < nummods; i++)
+	{
+		strcpy(modname, "[");
+		if (strlen(modnames[i]) < 16)
+		{
+			strcat(modname, modnames[i]);
+			for (int j=0; j < 15 - strlen(modnames[i]); j++)
```

Comment:
this loop could be a single call to memset() (with `' '` as char)

Reply:
I am not familiar with that - I also don't see the "harm" in how it's done.

Reply:
Oh wait... it's the code BELOW this comment, not above...  Still finding my footing here.

Now I get what you mean (2933 `strcat(modname, " ");` ), tho I still see no harm in how it's donw.